saleor/invoice/notifications.py

```diff
@@ -33,5 +33,11 @@ def send_invoice(
         "requester_app_id": app.id if app else None,
         **get_site_context(),
     }
-    manager.notify(NotifyEventType.INVOICE_READY, payload)  # type: ignore
+
+    channel_slug = None
```

Comment:
For invoices with removed orders we still sending multiple notifications. I'm not sure it's proper behavior.